src/lib.rs

```diff
@@ -120,23 +124,8 @@ pub fn clockboard(
         round(polygon, params.precision);
     }
 
-    let features: Vec<Feature> = polygons
-        .iter()
-        .map(|poly| Feature {
-            bbox: None,
-            geometry: Some(Geometry::from(poly)),
-            id: None,
-            properties: None,
-            foreign_members: None,
-        })
-        .collect();
-
-    let fc = FeatureCollection {
-        bbox: None,
-        features,
-        foreign_members: None,
-    };
-
+    let gc = geo::GeometryCollection::from_iter(polygons);
```

Comment:
That's looking a bit better! Thanks for helping us improve the API. 

There's one more [PR](https://github.com/georust/geojson/pull/165) I'd like to get merged before publishing these changes (so that you don't have to depend on a GH branch).

I'll ping people for a review...